src/reverse_offload/block_handle.hpp

```diff
@@ -130,7 +130,7 @@ class BlockHandleProxy {
   size_t num_blocks_{};
 };
 
-using BlockHandleProxyT = BlockHandleProxy<HIPAllocator>;
+using BlockHandleProxyT = BlockHandleProxy<HIPDefaultFinegrainedAllocator>;
```

Comment:
same comment as above, I think we want to try to limit this change to gfx90a platforms